src/components/Guilds/Sidebar/MemberActions.tsx

```diff
@@ -11,19 +12,21 @@ import {
   DropdownHeader,
 } from '../common/DropdownMenu';
 import { IconButton, Button } from '../common/Button';
-import dxIcon from '../../../assets/images/dxdao-icon.svg';
-
 import { useDetectBlur } from 'hooks/Guilds/useDetectBlur';
-import { useERC20 } from '../../../hooks/Guilds/contracts/useContract';
-import { ZERO_ADDRESS } from '../../../utils';
+import { shortenAddress } from '../../../utils';
 import { BigNumber } from 'ethers';
-import { useTransactions } from '../../../contexts/Guilds';
 import { useVotingPowerOf } from '../../../hooks/Guilds/ether-swr/useVotingPowerOf';
-
-const Icon = styled.img`
-  height: 1.1rem;
-  width: 1.1rem;
-`;
+import { useGuildConfig } from '../../../hooks/Guilds/ether-swr/useGuildConfig';
+import { useERC20Info } from '../../../hooks/Guilds/ether-swr/erc20/useERC20Info';
+import useENSAvatar from '../../../hooks/Guilds/ens/useENSAvatar';
+import { DEFAULT_ETH_CHAIN_ID } from '../../../provider/connectors';
+import Avatar from '../Avatar';
+import { formatUnits } from 'ethers/lib/utils';
+import { useVoterLockTimestamp } from '../../../hooks/Guilds/ether-swr/useVoterLockTimestamp';
+import moment from 'moment';
+import StakeTokensModal from '../StakeTokensModal';
+import { useTransactions } from '../../../contexts/Guilds';
+import { useERC20Guild } from '../../../hooks/Guilds/contracts/useContract';

```

Comment:
Would be nice to organise these imports and group them together some more

Reply:
I hope we can do this in a separate PR that changes the imports to absolute ones and implements import grouping through Prettier?